### ISSUE TITLE
Links to imgaug helpers in the README are broken

### ISSUE THREAD

**Issue by dniku** [albu/albumentations#368]
Links to `imgaug` helpers in https://github.com/albu/albumentations/blob/master/README.md are broken.

For example, the link for `IAAAdditiveGaussianNoise` points to

https://albumentations.readthedocs.io/en/latest/api/augmentations.html#albumentations.imgaug.transforms.IAAAdditiveGaussianNoise

instead of

https://albumentations.readthedocs.io/en/latest/api/imgaug.html#albumentations.imgaug.transforms.IAAAdditiveGaussianNoise

Basically, `augmentations.html` should be replaced with `imgaug.html`.

**Comment by ternaus** [comment]
Fixed.
